src/core/BufferGeometry.js

```diff
@@ -50,6 +49,13 @@ THREE.BufferGeometry.prototype = {
 
 	},
 
+	offsets: function () {
+
+		console.warn( 'THREE.BufferGeometry: .offsets has been renamed to .drawcalls.' );
+		return this.drawcalls.slice();
```

Comment:
Done.

> Seems like it will be less destructive.

If anything, it will be *more* destructive, which will force people to migrate!